package.json

```
@@ -41,7 +41,8 @@
     "prepare": "yarn run build && husky install",
     "preversion": "yarn install && yarn lint && yarn test",
     "version": "yarn docs-build && git add -A docs",
-    "postversion": "git push && git push --tags && npm publish"
+    "postversion": "git push && git push --tags && npm publish",
+    "watch": "watch 'yarn build' ./src/components --wait=3"
```

watch dep needs to be added to devDependencies 
```suggestion
    "watch": "watch 'yarn build' ./src --wait=3"
```

---

actually rollup already has a watch mode that can be used to rebuild the lib

---

Yes, this is my mistake. I actually didn't want to commit this one. I will remove this script